app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if command[0] == "mv" and command[2][-1] != "/":

```

Comment:
validate that command have 3 params
also it will does not work on windows: command[2][-1] != "/"